src/components/OktaSignInWidget/index.scss

```diff
@@ -7,9 +7,42 @@
     h3 {
       color: #595959;
     }
+    margin: 0;
   }
 
   #okta-sign-in.auth-container.main-container {
     color: #595959;
   }
 }
+
+.center-container {
+  margin: 100px auto 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+ /* Ensure the container is positioned relative for the overlay */
+.okta-sign-in-container {
+  position: relative;
+  display: inline-block;
+  width: auto; 
+  height: auto; 
+}
+
+#mint-okta-sign-in {
+  position: relative;
+}
+
+.nested-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.3);
```

Comment:
Definitely a `nit` -- but would you be open to considering a lighter color for the overlay?

Here's what this color looks like (using a _more_ opaque white instead of a _less_ opaque black)
 
```suggestion
  background-color: rgba(255, 255, 255, 0.6);
```

![Kapture 2024-09-24 at 09 05 41](https://github.com/user-attachments/assets/bae7192d-184c-4236-91b8-c432ef359817)